packages/compass-query-history/src/components/query/query.jsx

```diff
@@ -34,32 +55,39 @@ class Query extends PureComponent {
     const { attributes } = this.props;
 
     return (
-      <li key={index}>
-        <label
+      <div
+        className={queryAttributeStyles}
+        key={index}
+      >
+        <Label
           data-test-id="query-history-query-label"
-          className={styles.label}>{attrKey}</label>
+          className={labelStyles}
+          htmlFor={`query-history-query-attr-${attrKey}`}
+        >{attrKey}</Label>
         <Code
-          className={styles.code}
+          className={codeStyles}
+          id={`query-history-query-attr-${attrKey}`}
           data-test-id="query-history-query-code"
           language="javascript"
           copyable={false}
         >
           {formatQuery(attributes[attrKey])}
         </Code>
-      </li>
+      </div>
     );
   };
 
   render() {
     const { attributes } = this.props;
 
     return (
-      <ul
+      <button
         onClick={this.populateQuery}
-        className={styles.component}
-        data-test-id="query-history-query-attributes">
+        className={queryAttributesContainerStyles}
+        data-test-id="query-history-query-attributes"
+      >
         { Object.keys(attributes).map(this.renderAttr) }
-      </ul>
+      </button>
```

Comment:
Updated to fix the non-interactive html tag lint issue.
Before:
![Screen Shot 2022-07-06 at 3 45 01 PM](https://user-images.githubusercontent.com/1791149/177641672-b7112d67-42f5-4d15-aebc-2339ab45329c.png)

After:
![Screen Shot 2022-07-06 at 3 44 54 PM](https://user-images.githubusercontent.com/1791149/177641687-9ef5f09e-4a4d-4a4c-b770-24077b35b725.png)